packages/core/src/App/Containers/CashierModal/p2p-cashier.jsx

```diff
@@ -0,0 +1,7 @@
+import React  from 'react';
+import { WS } from 'Services';
+import P2P    from 'deriv-p2p';
+
+const P2PCashier = () => <P2P websocket_api={WS} />
```

Comment:
This will later be changed to create WS connection internally within P2P in case we use a different app ID, right? If yes, can you please put a comment there.